tablo/models.py

```diff
@@ -388,10 +353,10 @@ def _build_where_clause(self, where, count_only, **kwargs):
 
             if start_time:
                 if start_time == end_time:
-                    where_clause += ' AND {time_field} = %s'.format(time_field=layer_time_field)
+                    where_clause += ' AND {time_field} = %s::date'.format(time_field=layer_time_field)
```

Comment:
Will a `datetime` be too specific here? Should we convert `layer_time_field` from `datetime` to a date?

Reply:
I don't think we need that level of granularity. Also, I think it was causing an issue during import, and that's why I had to cast the date field to date.